internal/server/providers/manifest/provider_test.go

```
@@ -0,0 +1,257 @@
+	if !strings.Contains(err.Error(), "manifest yaml") {
+		t.Fatalf("error should mention manifest yaml, got %q", err.Error())
+	}
+}
+
+func TestProvider_Start_LoadsAndExposes(t *testing.T) {
+	path := writeManifest(t, sampleYAML)
+	p := New(path, WithInterval(time.Hour))
+	if err := p.Start(context.Background()); err != nil {
+		t.Fatalf("Start: %v", err)
+	}
+	defer p.Stop()
```

golangci-lint (2.12.2)

[error] Error return value of `p.Stop` is not checked

(errcheck)